core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -3131,8 +3131,8 @@ class ViewMapping<
 
   using handle_type = typename ViewDataHandle<Traits>::handle_type;
 
-  handle_type m_impl_handle;
-  offset_type m_impl_offset;
+  handle_type m_impl_handle = nullptr;
+  offset_type m_impl_offset{};
```

Comment:
Why are these changes necessary?

Reply:
I think the change I need is only `handle_type m_impl_handle = nullptr;`.

These changes are related to your comment below. I defaulted the constructor, so I wanted to provide proper initialization of the members.